sky/clouds/service_catalog/common.py

```diff
@@ -522,8 +528,11 @@ def get_instance_type_for_accelerator_impl(
         fuzzy_candidate_list = []
         if len(fuzzy_result) > 0:
             for _, row in fuzzy_result.iterrows():
+                acc_cnt = row['AcceleratorCount']
+                acc_count_display = (int(acc_cnt)
+                                     if acc_cnt.is_integer() else acc_cnt)
```

Comment:
Printing a float may cause a long output, should we limit the digits?
```suggestion
                                     if acc_cnt.is_integer() else f'{acc_cnt:.2f}')
```